wfe2/wfe.go

```diff
@@ -1551,29 +1552,15 @@ func (wfe *WebFrontEndImpl) NewOrder(
 	}
 }
 
-// Order is used to retrieve a existing order object
-func (wfe *WebFrontEndImpl) Order(ctx context.Context, logEvent *web.RequestEvent, response http.ResponseWriter, request *http.Request) {
-	fields := strings.SplitN(request.URL.Path, "/", 3)
+// GetOrder is used to retrieve a existing order object
+func (wfe *WebFrontEndImpl) GetOrder(ctx context.Context, logEvent *web.RequestEvent, response http.ResponseWriter, request *http.Request) {
+	fields := strings.SplitN(request.URL.Path, "/", 2)
 	// If there are less than 2 fields there can't be both an account ID and an
 	// order ID so the path is invalid
 	if len(fields) < 2 {
```

Comment:
Super nit: when `len(fields)` could be 2 or 3 this made sense, now it'd probably be a bit less confusing as `len(fields) != 2`.